src/lib/components/layout/Grid/GridSpan.jsx

```diff
@@ -0,0 +1,85 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { generateResponsiveCustomProperties } from './helpers';
+import styles from './Grid.scss';
+
+export const GridSpan = (props) => {
```

Comment:
I would prefer the destructuring syntax:

```js
export const GridSpan = ({
    children,
    columns,
    id,
    rows,
    ...other
  }) => {
```

I know it is not used elsewhere, but I find it easier to read. In fact I think we should update other files to use it as well (not in this PR obviously).

Reply:
> In fact I think we should update other files to use it as well (not in this PR obviously).

Sure, there is an issue for that: #113.

Reply:
And thank you for pointing it out, I missed it somehow this time.